crates/modelardb_common/src/test/mod.rs

```diff
@@ -28,12 +28,18 @@ use crate::metadata::model_table_metadata::ModelTableMetadata;
 use crate::schemas::COMPRESSED_SCHEMA;
 use crate::types::{ArrowTimestamp, ArrowValue, ErrorBound, TimestampArray, ValueArray};
 
+/// Expected size of the multivariate data produced in the tests.
+pub const MULTIVARIATE_DATA_SIZE: usize = 1436344;
+
 /// Expected size of the compressed data buffers produced in the tests.
```

Comment:
This was not changed in this PR but it should be `uncompressed data buffers produced` instead.

Reply:
Fixed.